Amplify/Amplify.swift

```diff
@@ -79,7 +79,7 @@ public class Amplify {
     ///
     /// - Parameter plugin: The plugin to add
     /// - Tag: Amplify.add_plugin
-    public static func add<P: Plugin>(plugin: P) throws {
+    public static func add(plugin: some Plugin) throws {
```

Comment:
We might want to exclude this `cyclomatic_complexity` rule for this method?

Reply:
good suggestion, updated.